src/main/java/com/igot/cb/transactional/elasticsearch/service/EsUtilServiceImpl.java

```diff
@@ -45,4 +59,22 @@ public Boolean updateUserOrgCustomFields(String userId, String orgId, List<Map<S
             return false;
         }
     }
+
+    public EsResponse saveObjectInIgotES(Object doc, String indexName, String docType, String docId) {
+        if (doc == null)
+            return EsResponse.builder().success(false).message("Document object is null").build();
+        if (docId == null || docId.trim().isEmpty())
+            return EsResponse.builder().success(false).message("Document ID must not be null or empty").build();
+
+        try {
+            Map<String, Object> docMap = objectMapper.convertValue(doc, Map.class);
+            IndexRequest request = new IndexRequest(indexName, docType).id(docId).source(docMap);
+            igotESClient.index(request, RequestOptions.DEFAULT);
+            log.info("Document indexed successfully in IGOT ES, index [{}], type [{}], id [{}]", indexName, docType, docId);
+            return EsResponse.builder().success(true).message("Document indexed successfully").documentId(docId).build();
+        } catch (Exception e) {
+            log.error("Error indexing document in IGOT ES, index [{}]: {}", indexName, e.getMessage(), e);
+            return EsResponse.builder().success(false).message("Error indexing document: " + e.getMessage()).build();
+        }
```

Comment:
this has to be upsert query against ES -- which does either index or update against ES 
refer the ES util classes from sunbird-cb-workflow service repos